ansible/install-gluster-health-report.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: gluster_servers
+  gather_facts: no
+  remote_user: root
+
+  tasks:
+  - name: Downloading pip2 from bootstrap.pypa
```

Comment:
easy_install can be used if pip is not available https://docs.ansible.com/ansible/latest/modules/easy_install_module.html

Reply:
CentOS 7.7 and latest Fedora will have Python 3 and pip by default.

Reply:
I feel that changing it to directly to use easy_install will make more sense as easy_install would be available on all distributions by default.